apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -317,10 +317,10 @@ export const PoolTable = ({ pools }: PoolTableProps) => {
           // if groups identical then sort by level
           // level sorting adjusted to always be ascending regardless of whether group sort is A-Z or Z-A
```

Comment:
Should probably remove this outdated comment, too.

Reply:
Removed in [remove outdated comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6453/commits/ed001466a21fc1da69ef487ce940c2b4be8dfe97)